lib/api/src/modules/addons.ts

```diff
@@ -1,10 +1,21 @@
 import { ReactElement } from 'react';
-
 import { WindowLocation } from '@reach/router';
+import deprecate from 'util-deprecate';
+import dedent from 'ts-dedent';
+
 import { ModuleFn } from '../index';
 import { Options } from '../store';
 import { isStory } from '../lib/stories';
 
+const warnDisabledDeprecated = deprecate(
+  () => {},
+  dedent`
+    Use 'parameters.key.disable' instead of 'parameters.key.disabled'.
+    
+    https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-disabled-parameter
+  `
+);
+
```

Comment:
I didn't checkout to run this but I believe it would work 🤔 

This way the message is straight to the point with no confusion

```suggestion
const warnDisabledDeprecated = (paramKey) => deprecate(
  () => {},
  dedent`
    Use 'parameters.${paramKey}.disable' instead of 'parameters.${paramKey}.disabled'.
    
    https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-disabled-parameter
  `
)();
```